templates/create-daml-app/daml.yaml.template

```diff
@@ -14,3 +14,6 @@ sandbox-options:
 - --wall-clock-time
 - --ledgerid=__PROJECT_NAME__-sandbox
 start-navigator: false
+codegen:
+  js:
+    output-directory: ui/daml.js
```

Comment:
Don't we need to add the `npm-scope` here as well?

Reply:
This defaults to `daml.js`.

Reply:
I see. I guess it would still make sense to call it out explicitly in order to give our users a hint that they can change this if they wish. What do you think?

Reply:
Makes sense, I'll add it.

Reply:
Cool. Thank you.